doc/api/cli.md

```diff
@@ -2018,6 +2018,31 @@ On a machine with 2 GiB of memory, consider setting this to
 $ node --max-old-space-size=1536 index.js
 ```
 
+### `--max_semi_space_size=SIZE` (in megabytes)
+
+Sets the maximum
+[semi-space](https://www.memorymanagement.org/glossary/s.html#semi.space)
+size for V8's [scavenge](https://v8.dev/blog/orinoco-parallel-scavenger)
+garbage collector in MiB.
+Increasing the semi-space size by 1MiB will
```

Comment:
Added reason and reference for clarity.
Formatted size (with trailing space before unit) for consistency with `cli.md`.

---
```suggestion
As there is more than one semi-space ([#42950](https://github.com/nodejs/node/discussions/42950)),
increasing the semi-space size by 1 MiB will
```

Reply:
I used the related [source code](https://chromium.googlesource.com/v8/v8.git/+/refs/heads/main/src/heap/heap.cc#328) from V8 to explain the tripled relationship between memory consumption and semi-space size. I think this is more convincible.

Reply:
Agreed. But I still think it's a little unclear in the new changes and I will start a new review to address that, hopefully.